demo/docroot/.ht.router.php

```diff
@@ -26,8 +26,15 @@
 
 $url = parse_url($_SERVER['REQUEST_URI']);
 if (file_exists(__DIR__ . $url['path'])) {
-  // Serve the requested resource as-is.
-  return FALSE;
+
+  if (strpos($url['path'], '.js') !==  FALSE) {
```

Comment:
Hmm, I see that we have to make this change but this also means that the API wouldn't work with a standard installation of Drupal.

Reply:
We should have documented this, and yes that's really sad. This is due to CORS loading the JS file. This is _just_ a problem when you run Drupal on a different domain than the UI itself. Usescases for that:

* Advanced setup: When you have an external backend, and a separate Drupal. In this setup you might have easy time controlling that behaviour.
* Development mode for the admin UI. In this mode you might or might not have control. We could ship with a custom route which forwards these files, so that they would get CORS headers automatically.

Sadly this workaround would completely disappear when we could build the files.